src/topNavigation/components/_topNavigationFavouritesIcon.scss

```diff
@@ -7,15 +7,15 @@
     position: absolute;
     left: $spacing-xs;
     top: -$spacing-xs;
-    background-color: $color-gold;
+    background-color: $color-info;
```

Comment:
I feel that gold worked better here and it was only the icons on the map that were supposed to be changed, but if Matias instructed you to change this as well, then this is fine.

Reply:
I understood that he wanted "to get rid of the gold (shitty-brown) as it was ugly" :D We can ask this in sprint review and change later if needed.. I make a card to backlog..